editor/components/block-list/with-hover-areas.js

```diff
@@ -34,13 +35,14 @@ const withHoverAreas = createHigherOrderComponent( ( WrappedComponent ) => {
 		}
 
 		onMouseMove( event ) {
+			const { isRtl } = this.props;
 			const { width, left, right } = this.container.getBoundingClientRect();
 
 			let hoverArea = null;
 			if ( ( event.clientX - left ) < width / 3 ) {
-				hoverArea = 'left';
+				hoverArea = isRtl ? 'right' : 'left';
```

Comment:
I think it should be called isRTL, similar to findDOMNode.

Would it make sense to introduce a helper like:
const direction = ( value) => {
    if ( ! isRTL) {
        return value;
    }
    return value === ‚right’ ? ‚left’ : ‚right’;
};

My mobile keyboard is a bit limited 😭

Reply:
I personally don't think adding this helper will help. I like dumb code :)

I'll update to `isRTL`